openedx/features/journals/templates/journals/learner_dashboard/journal_dashboard.html

```diff
@@ -0,0 +1,135 @@
+<%page expression_filter="h"/>
```

Comment:
NOTE TO @bderusha : 

I left off here but wanted to at least get the current feedback in.  I will pickup with my review here tomorrow